src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -9,7 +9,7 @@
 from typing import Tuple
 from typing import Union
 
-from .._version import __version__ as VERSION
+from .. import __version__ as VERSION
```

Comment:
I don't think that the project internals should access version through the package init. They should just get it from the `_version` as before. And `_version` should guard against import errors internally.